content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Plans and quotas'
+metaTitle: 'Plans and quotas'
+metaDescription: 'The Prisma Data Platform provides a set of monthly plans that include quotas for the number of collaborators and environments per project, as well as a quota for the usage of the Prisma Data Proxy (in CPU milliseconds).'
+---
+
+<TopBlock>
+
+The Prisma Data Platform provides a set of monthly plans that include quotas for:
+
+- number of collaborators per project
+- number of environments per project
+- Prisma Data Proxy usage (in CPU milliseconds) per project
+
+</TopBlock>
+
+## About plans and quotas
+
+Plans in the Prisma Data Platform are designed with the goal to help you scale your projects as the need for team members, environments, and Data Proxy usage grows.
+
+Because of this, a single Prisma Data Platform plan applies for a single project.
+
+We want to give you the option to create projects freely and you can then decide which project you want to upgrade to a paid plan. You can have many Prisma projects and you can upgrade only several of them to a paid plan.
+
+<Admonition>
+
+**Account-wide plans**<br /><br />
+The notion of an account-wide plan that includes a certain number of projects or provides access to specific features does not exist in the Prisma Data Platform.<br />
+
+</Admonition>
+
+## The first plan of each project
+
+- If a project is created or added in the Prisma Data Platform before the [General Availability](/about/prisma/platform-releases#maturity-levels) of the platform, it starts on the [**Early Adopter**](/concepts/data-platform/billing/early-adopter-plan) plan.
+- If project is created after General Availability, it starts on the **Free** plan.
+
+## Comparison of plans
+
+For plan pricing details, go to our [Pricing](https://cloud.prisma.io/pricing) page.
+
+|                                               | **Early Adopter**                                                              | **Free**                                                 | **Scale**                                                                                                                                                                 | **Enterprise**                                              |
```

Comment:
I'll go with the group about whether we want to duplicate the pricing details in the Docs on top of having them elsewhere on the website. I'm only afraid about having drifts between the two sources of content over time, which can be highly problematic past a point if someone uses the "wrong" source of information.

Reply:
@tanberry and I shared this concern and we want to avoid pricing maintenance too. This is why actual pricing/support are not really included, only factual information about the features and how the plans compare based on those.

I've decided to add it anyway because of these:

- This is the only opportunity to also compare the EA plan (until we remove it)
- I've made several iterations to clarify the **Quota** and **Overages** bits for collaborators, environments, data proxy usage. If these make sense to also make it to the Pricing page, then that'll become a moot point here.

They are not really that strong points, but as a novice to PDP I found it useful to have that overview.

Reply:
OK, let's leave that as is then and revisit if it becomes a potential problem